client/src/ui/LoadStatus.res

```diff
@@ -0,0 +1,21 @@
+// LoadStatus is a value to represent that there is or was a loading action.
+
+// This is deliberately simple, same as SaveStatus
+//
+// It is mostly used to indicate in the UI whether something is loading. If something
+// has finished loading, it is Complete. If it is complete, it will have a value: that
```

Comment:
is "Complete" here meant to match `Success` below, or is that a separate term to keep in mind?

Reply:
Typo/outdated, thanks!